packages/meditrak-server/src/routes/importSurveyResponses/assertCanImportSurveyResponses.js

```diff
@@ -57,3 +57,29 @@ export const assertCanImportSurveyResponses = async (
 
   return true;
 };
+
+export const assertSurveyResponseBatchPermissions = async (
```

Comment:
To use the existing `assertCan{{verb}}{{noun}}` assertion naming pattern

```suggestion
export const assertCanSubmitSurveyResponses = async (
```